app/src/main/java/com/kickstarter/viewmodels/PaymentMethodsViewModel.kt

```diff
@@ -140,9 +166,24 @@ class PaymentMethodsViewModel(environment: Environment) : ViewModel(), PaymentMe
         )
 
         compositeDisposable.add(
-            this.refreshCards
-                .switchMap { getListOfStoredCards() }
-                .subscribe { this.cards.onNext(it) }
+            this.refreshCards.subscribe {
```

Comment:
question here, I assume this piece is for demonstration purposes, because this piece over here be just exchanging within the switch map `getListOfStoredCards()` over  `apolloClient.getStoredCards()`, let me know if I'm missing something!